Sources/Core/iOS/ActivityIndicatorViewObserver.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ActivityIndicatorViewObserver.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
+//
+//
+
+import UIKit
+
+public protocol ActivityIndicatorViewAnimationInterface {
+    func startAnimating()
+    func stopAnimating()
+}
+
+extension UIActivityIndicatorView: ActivityIndicatorViewAnimationInterface {}
+
+/**
+ An OperationObserverType that can update the state of an associated `UIActivityIndicatorView`
+ when the observed operation starts and finishes.
+ 
+ - note: Any type conforming to `ActivityIndicatorViewAnimationInterface` can be provided as 
+         the activity indicator.
+ */
+public class ActivityIndicatorViewObserver: OperationDidStartObserver, OperationDidFinishObserver {
+
+    private let activityIndicator: ActivityIndicatorViewAnimationInterface
```

Comment:
Change this to `private weak var activityIndicator: ActivityIndicatorViewAnimationInterface`

Essentially some other controller/view/nib should be owning the activity indicator view.